java/README.rst

```diff
@@ -0,0 +1,202 @@
+#####################
+GRPC-Java OpenTracing
+#####################
+
+============
+Installation
+============
+
+This package is available on Maven Central and can be added to your project as follows:
+
+**Maven**
+
+.. code-block::
+
+	<dependencies>
+		<dependency>
+			<groupId>io.opentracing.contrib.grpc</groupId>
```

Comment:
no need to resolve now, but we should figure out if grpc wants to host these artifacts themselves